bin/release_latest.js

```diff
@@ -32,10 +32,10 @@ gh.readFile(config.filename, { ref: config.branches[0] }, function (err, data) {
     }
   }
   var version = config.version
-  var macLine = `/mapeo/latest/mac https://github.com/digidem/mapeo-desktop/releases/download/v${version}/Installar_Mapeo_v${version}_mac.dmg 302`
-  var windowsLine = `/mapeo/latest/windows https://github.com/digidem/mapeo-desktop/releases/download/v${version}/Installar_Mapeo_v${version}_win-x64.exe 302`
-  var win32Line = `/mapeo/latest/win32 https://github.com/digidem/mapeo-desktop/releases/download/v${version}/Installar_Mapeo_v${version}_win-ia32.exe 302`
-  var linuxLine = `/mapeo/latest/linux https://github.com/digidem/mapeo-desktop/releases/download/v${version}/Installar_Mapeo_v${version}_linux.deb 302`
+  var macLine = `/mapeo/latest/mac https://github.com/digidem/mapeo-desktop/releases/download/v${version}/Install_Mapeo_v${version}_mac.dmg 302`
+  var windowsLine = `/mapeo/latest/windows https://github.com/digidem/mapeo-desktop/releases/download/v${version}/Install_Mapeo_v${version}_win-x64.exe 302`
+  var win32Line = `/mapeo/latest/win32 https://github.com/digidem/mapeo-desktop/releases/download/v${version}/Install_Mapeo_v${version}_win-ia32.exe 302`
+  var linuxLine = `/mapeo/latest/linux https://github.com/digidem/mapeo-desktop/releases/download/v${version}/Install_Mapeo_v${version}_linux.deb 302`
```

Comment:
@gmaclennan is it okay to remove this file as a whole? Just realized that it's not used anywhere, so not sure if it was part of a some local flow or something else

Reply:
Yes, it can be removed. Although this was the script that updated the website download links, which is why @ErikSin needed to manually update the website for the last release. I have now updated our website to use the permalinks for the latest releases which are provided by our [mapeo-releases-worker](https://github.com/digidem/mapeo-releases-worker)